### ISSUE TITLE
Jodit does not work in popout windows

### ISSUE THREAD

**Issue by marchaos** [xdan/jodit#150]
I'm using Jodit in my main app as well as a popout window. I'm passing in ownerDocument and ownerWindow for the document and window of the popout.

Jodit works nicely in the main app, and displays ok in the popout, but as soon as you attempt to do something like drop an image, you get an error at:

```
if (!Dom.isNode(node, this.win)) {
    throw new Error('Parameter node most be instance of Node');
}
```
in insertNode, Selection.ts:481

this.win refers to the window of the popout (good), but the node has been created via the document of the main window. insertNode in this case was being called from insertImage, which uses ```this.jodit.create.inside.element('img')```. ```this.jodit``` is a Jodit object and it inherits create from Panel.ts. The main issue is that Panel is expecting a jodit object so that it can obtain the ownerDocument and ownerWindow, which is not possible since we cannot pass ourselves "this" from Jodit constructor all the way up to Panel.ts. 

**Jodit Version:**  3.2.28

**Expected behavior:**

Jodit should work in a popout

**Actual behavior:**

Errors since the ownerDocument and ownerWindow do not reference the correct window in the Create instance.

**Comment by marchaos** [comment]
As a hack, I've added 

```
this.create = new Create(
    this.ownerDocument,
    this.editorDocument
);
```
to Jodit.constructor which fixes the above issue, but there are other problems in the popout that are not present when in the main document. I'm investigating those now.

**Comment by xdan** [comment]
Fixed in 3.2.42
